### PR TITLE
neonpostgres add live test

### PR DESCRIPTION
**Pull request by jliusan**
add live test

### CODE REVIEW

File: sdk/resourcemanager/neonpostgres/armneonpostgres/neonpostgres_live_test.go [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+	pager := clientFactory.NewOperationsClient().NewListPager(nil)
+	for pager.More() {
+		page, err := pager.NextPage(ctx)
+		if err != nil {
+			log.Fatalf("failed to advance page: %v", err)
+		}
+		for _, v := range page.Value {
+			fmt.Println("page value", v)
+		}
+		page = armneonpostgres.OperationsClientListResponse{
```
Comment (tadelesh) [inline sdk/resourcemanager/neonpostgres/armneonpostgres/neonpostgres_live_test.go:89]: why you do this?

File: sdk/resourcemanager/neonpostgres/armneonpostgres/neonpostgres_live_test.go [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+	cred, err := azidentity.NewDefaultAzureCredential(nil)
+	if err != nil {
+		log.Fatalf("failed to obtain a credential: %v", err)
+	}
+	clientFactory, err := armneonpostgres.NewClientFactory(testsuite.subscriptionId, cred, testsuite.options)
+	if err != nil {
+		log.Fatalf("failed to create client: %v", err)
+	}
+
+	_, err = clientFactory.NewOrganizationsClient().BeginCreateOrUpdate(testsuite.ctx, testsuite.resourceGroupName, testsuite.organizationName, armneonpostgres.OrganizationResource{
```
Comment (tadelesh) [inline sdk/resourcemanager/neonpostgres/armneonpostgres/neonpostgres_live_test.go:121]: i think it should not be placed in prepare. it is a test case for create or update.

File: sdk/resourcemanager/neonpostgres/armneonpostgres/neonpostgres_live_test.go (hunk 1 above)
Comment (jliusan) [reply sdk/resourcemanager/neonpostgres/armneonpostgres/neonpostgres_live_test.go]: my mistake, have fixed

File: sdk/resourcemanager/neonpostgres/armneonpostgres/neonpostgres_live_test.go [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+	_, err := testutil.DeleteResourceGroup(testsuite.ctx, testsuite.subscriptionId, testsuite.cred, testsuite.options, testsuite.resourceGroupName)
+	testsuite.Require().NoError(err)
+	testutil.StopRecording(testsuite.T())
+}
+
+func TestNeonpostgresTestSuite(t *testing.T) {
+	suite.Run(t, new(NeonpostgresTestSuite))
+}
+
+func (testsuite *NeonpostgresTestSuite) TestOptionList() {
```
Comment (tadelesh) [inline sdk/resourcemanager/neonpostgres/armneonpostgres/neonpostgres_live_test.go:68]:
```suggestion
func (testsuite *NeonpostgresTestSuite) TestOperationsList() {
```

File: sdk/resourcemanager/neonpostgres/armneonpostgres/neonpostgres_live_test.go [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+	clientFactory, err := armneonpostgres.NewClientFactory(testsuite.subscriptionId, cred, testsuite.options)
+	testsuite.Require().NoError(err)
+	pager := clientFactory.NewOperationsClient().NewListPager(nil)
+	for pager.More() {
+		_, err := pager.NextPage(ctx)
+		testsuite.Require().NoError(err)
+	}
+}
+
+func (testsuite *NeonpostgresTestSuite) TestCreateOrUpdate() {
```
Comment (tadelesh) [inline sdk/resourcemanager/neonpostgres/armneonpostgres/neonpostgres_live_test.go:81]:
```suggestion
func (testsuite *NeonpostgresTestSuite) TestOrganizationsCreateOrUpdate() {
```

**Review by tadelesh — APPROVED** [review]

**Review by tadelesh — APPROVED** [review]

**Pull request merged**